eagle-jpm/eagle-jpm-mr-history/src/main/java/org/apache/eagle/jpm/mr/history/MRHistoryJobApplicationProvider.java

```diff
@@ -35,7 +40,16 @@ public MRHistoryJobApplication getApplication() {
     }
 
     @Override
-    public Optional<HealthCheck> getAppHealthCheck(Config config) {
+    public Optional<HealthCheck> getManagedHealthCheck(Config config) {
         return Optional.of(new MRHistoryJobApplicationHealthCheck(config));
     }
+
+    @Override
+    public Optional<List<Service>> getSharedServices(Config envConfig) {

```

Comment:
Change#2: move `MRHistoryJobDailyReporter ` registry to `MRHistoryJobApplicationProvider` from `ServerApplication` 

@qingwen220 @wujinhu